editor/src/clj/editor/updater.clj

```diff
@@ -337,14 +337,14 @@
         install-dir (.getCanonicalFile
                       (if-let [path (system/defold-resourcespath)]
                         (case os
-                          "darwin" (io/file path "../../")
+                          "macos" (io/file path "../../")
```

Comment:
This is a part where I'd say that the editor doesn't need a copy of Platform.java